Runtime/Model/Database/BacktraceDatabaseRecord.cs

```diff
@@ -11,7 +9,7 @@ namespace Backtrace.Unity.Model.Database
     /// <summary>
     /// Single record in BacktraceDatabase
     /// </summary>
-    public class BacktraceDatabaseRecord : IDisposable
```

Comment:
Disposable will clean native resources that we don't allocate here. Because of that this is additional cost that we were paying for removing objects that exists only in managed code.

In this case there is no reason to use Idisposable, because we didn't allocate any native resources